modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -0,0 +1,151 @@
+
+For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. This helps in projecting the binding data into the container path specified in the secondary workload of the `ServiceBinding` CR, for example, when you do not want the binding data inside a pod.
+
+{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location. This option is available only for the `binding.operators.coreos.com` API group when the binding data is projected as environment variables.
+
+=== Configuring the custom location of the container path
+Consider a secondary workload CR, which is not compliant with the PodSpec, with containers at the `spec.containers` path:
+
+.Example: Secondary workload CR
+[source,yaml]
+----
+apiVersion: "operator.sbo.com/v1"
+kind: SecondaryWorkload
+metadata:
+    name: secondary-workload
+spec:
+    containers:
+    - name: hello-world
+      image: quay.io/baijum/secondary-workload:latest
+      ports:
+      - containerPort: 9966
+----
+
+The following example shows how to configure the value of where the `spec.containers` path is by specifying it in the `ServiceBinding` CR and bind this path at a `spec.application.bindingPath.containersPath` custom location:
+
+.Example: `ServiceBinding` CR with the `spec.containers` path in a custom location
+[source,yaml]
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+spec:
+    services:
+    - group: postgres-operator.crunchydata.com
+      version: v1beta1
+      kind: PostgresCluster
+      name: hippo
+      id: postgresDB
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-pguser-hippo
+      id: postgresSecret
+    application: <1>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    application: <2>
+      name: secondary-workload
+      group: operator.sbo.com
+      version: v1
+      resource: secondaryworkloads
+      bindingPath:
+        containersPath: spec.containers <3>
+----
+<1> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
+<2> Secondary workload, which is not compliant with the PodSpec.
+<3> Custom location of the container path.
+
+After you specify the location of the container path, {servicebinding-title} generates the binding data, which becomes available in the container path specified in the secondary workload of the `ServiceBinding` CR.
+
+The following example shows the `spec.containers` path, which is updated  with the `envFrom` and `secretRef` fields:
```

Comment:
```suggestion
The following example shows the `spec.containers` path with the `envFrom` and `secretRef` fields:
```
The passive voice in the relative clause at the end makes this a bit confusing. What or who is actually updating these fields? It's not clear to me. My edit might help, but maybe it changes the meaning too much

Reply:
The fields are updated by the Service Binding Operator. But I like the revision so going with your suggestion.